docker-compose.yml

```diff
@@ -115,13 +137,20 @@ services:
     caddy:
         image: caddy:latest
         ports:
-          - "80:80"
-          - "443:443"
+            - "80:80"
+            - "443:443"
+        networks:
+            - caddy
+            - zammad
```

Comment:
Does caddy also need access to the `zammad` network? I would assume just `caddy` would be sufficient.